nio/schemas.py

```diff
@@ -1213,6 +1213,32 @@ class Schemas(object):
         ],
     }
 
+    tags = {
+        "type": "object",
+        "properties": {
+            "type": {"type": "string"},
+            "content": {
+                "type": "object",
+                "properties": {
+                    "tags": {"type:": "object"},
+                    "properties": {
+                        "order": {"type": "float"}
+                    },
+                    "required": [
```

Comment:
Because the properties are in the wrong place this ended up in the wrong place as well, also there is no `type` property.